SpatialGDK/Source/SpatialGDKFunctionalTests/Private/GenerateTestMapsCommandlet.cpp

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#if WITH_EDITOR
+
+#include "GenerateTestMapsCommandlet.h"
+#include "Engine/ExponentialHeightFog.h"
+#include "Engine/SkyLight.h"
+#include "Engine/StaticMeshActor.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "FileHelpers.h"
+#include "GameFramework/PlayerStart.h"
+#include "SpatialGDKFunctionalTests/Public/SpatialCleanupConnectionTest.h" //didn't spot this during review, but I would argue this one is "SpatialGDK" as well
+#include "SpatialGDKFunctionalTests/Public/SpatialTest1x2GridSmallInterestWorkerSettings.h"
+#include "SpatialGDKFunctionalTests/Public/Test1x2WorkerSettings.h"
+#include "SpatialGDKFunctionalTests/Public/Test2x2WorkerSettings.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/CrossServerAndClientOrchestrationTest/CrossServerAndClientOrchestrationTest.h"
```

Comment:
Holy includes batman

Reply:
Yeah, an argument for splitting it into multiple files, probably one per test map.

Reply:
Yeah this is monolithic and not gonna be good for long term maintenance I believe. I think we should look at one per test map. Also, I was under the impression the design would utilise dynamic composition of tests / test map. Such that you define a test and it says "I want a 2x2 map" and calls a generate function to create the map and place itself in. This seems to be helper functions to create specific maps with all tests placed in. Are we sure about this design in the long run?

Reply:
Oho, second time this has come up basically.
I think the main thing blocking this is the fear of tests taking too long. I will probably just test this as well.